### PR TITLE
Split up wl seat

### PR DESCRIPTION
**Pull request by wmww**

### CODE REVIEW

File: src/server/frontend/wayland/wl_keybaord.h [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Authored by: Christopher James Halse Rogers <christopher.halse.rogers@canonical.com>
+ */
+
+#ifndef WL_KEYBOARD_H
```
Comment (AlanGriffiths) [inline src/server/frontend/wayland/wl_keybaord.h]: I may have missed this elsewhere, but we start our include guards with "MIR_", in this case it should be "MIR_FRONTEND_WL_KEYBOARD_H_"

Comment (wmww) [reply src/server/frontend/wayland/wl_keybaord.h]: I'll fix that.

Comment (wmww) [reply src/server/frontend/wayland/wl_keybaord.h]: hold on, I did it wrong.

Comment (wmww) [reply src/server/frontend/wayland/wl_keybaord.h]: fixed.

**Pull request merged**